### PR TITLE
teuthology-suite: Add job count to priority error msg.

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Don't let users guess the job count.

Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

### CODE REVIEW

**Review by varshar16 — APPROVED** [review]

File: teuthology/suite/run.py [hunk 1]
```diff
@@ -505,7 +505,9 @@ def schedule_jobs(self, jobs_missing_packages, jobs_to_schedule, name):
 
     def check_priority(self, jobs_to_schedule):
         priority = self.args.priority
-        msg='''Use the following testing priority
+        msg=f'''Error: Unable to schedule {jobs_to_schedule} jobs with priority {priority}.
```
Comment (kshtsk) [inline teuthology/suite/run.py:508]: Because this message goes to exception it is not needed to be prefixed with "Error: ", please remove it.

Comment (sebastian-philipp) [reply teuthology/suite/run.py]: done

**Comment by kshtsk** [comment]
@susebot run deploy

**Review by kshtsk — APPROVED** [review]

File: teuthology/suite/run.py [hunk 2]
```diff
@@ -505,7 +505,9 @@ def schedule_jobs(self, jobs_missing_packages, jobs_to_schedule, name):
 
     def check_priority(self, jobs_to_schedule):
         priority = self.args.priority
-        msg='''Use the following testing priority
+        msg=f'''Unable to schedule {jobs_to_schedule} jobs with priority {priority}.
+
+Use the following testing priority
```
Comment (kshtsk) [inline teuthology/suite/run.py:510]: Maybe add here a colon (:)?

**Pull request merged**